runtime/tests/checker/string_test.go

```diff
@@ -281,7 +281,7 @@ func TestCheckStringEncodeHex(t *testing.T) {
 	t.Parallel()
 
 	checker, err := ParseAndCheck(t, `
-        let x = String.encodeHex([1 as UInt8, 2, 3, 0xCA, 0xDE])
+        let x = String.encodeHex([1, 2, 3, 0xCA, 0xDE] as [UInt8])
```

Comment:
Could this now be even further simplified by removing the explicit type annotation?

```suggestion
        let x = String.encodeHex([1, 2, 3, 0xCA, 0xDE])
```

Reply:
oh right, I implemented this PR before adding type inference to function arguments 😄  So this wasn't working back then. But should be working now. Good catch!